demisto_sdk/commands/run_unit_tests/unit_tests_runner.py

```diff
@@ -141,19 +142,25 @@ def unit_test_runner(file_paths: List[Path], verbose: bool = False) -> int:
                     / "conftest.py",
                     integration_script.path.parent / "conftest.py",
                 )
+
+                logger.info(
+                    f"Running test for {integration_script.path} using {docker_image=} with {test_docker_image=}"
```

Comment:
```suggestion
                    f"Running test for {'/'.join(integration_script.path.parts[-3:])} using {docker_image=} with {test_docker_image=}"
```
Shorter messages, we don't need the whole path (please test that my suggestion didn't break it)

Reply:
changed to use the relative path